src/modules/UI/components/Header/Component/WalletSelector.ui.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React, {Component} from 'react'
+import type {State} from '../../../../ReduxTypes'
+import {TextAndIconButton} from '../../Buttons/TextAndIconButton.ui'
+import * as Constants from '../../../../../constants/indexConstants'
+import { CryptoExchangeSceneStyle } from '../../../../../styles/scenes/CryptoExchangeSceneStyles'
+
+export type StateProps = {
+  title: string
+}
+
+export type DispatchProps = {
+  onPress: () => any
+}
+
+type Props = StateProps & DispatchProps
+
+export default class WalletSelector extends Component<Props, State> {
+  render () {
+    return <TextAndIconButton
+      style={CryptoExchangeSceneStyle.flipWrapper.walletSelector}
+      icon={Constants.KEYBOARD_ARROW_DOWN}
+      iconType={Constants.MATERIAL_ICONS}
+      onPress={this.props.onPress}
+      title={this.props.title}
+      blat={'WalletSelectorConnector'}
```

Comment:
what is this?

Reply:
lol, forgot to remote it, used it to help me find the the module in the react stack when debugging :)
Will remove it